libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -63,6 +63,35 @@ export class StripeManager {
     return customer;
   }
 
+  /**
+   * Updates a customer record
+   */
+  async updateCustomer(
+    customerId: string,
+    params?: Stripe.CustomerUpdateParams
+  ) {
+    return await this.client.customersUpdate(customerId, params);
+  }
+
+  /**
+   * Create customer stub account
+   */
+  async createPlainCustomer(args: { email?: string; taxAddress?: TaxAddress }) {
+    if (args.taxAddress) {
+      return await this.client.customersCreate({
```

Comment:
Optional change. Could a similar approach be used as seen in `stripeManager.invoicePreview`, where the request object sent to the StripeClient call, is built up over depending on what data is available?

Functionality of `createPlainCustomer` is likely to expand in future, and following the suggested approach makes it easier to build out. But since it's not currently necessary, it can be picked up when needed, so that's why this is an optional change

Reply:
Totally aligned. I think I will keep it for now though because a lot of areas still need updating and when we refine it, might end up being easier as we'll have a better idea of what data we want to pass.